system-components/package.json

```diff
@@ -44,6 +44,7 @@
     "roots": [
       "<rootDir>"
     ],
+    "testURL": "http://localhost",
```

Comment:
Jest `23.5.0` came out a few days ago which resolved the need to have this in the config. But it's also harmless to have.